lib/plug/mailbox_preview.ex

```diff
@@ -53,6 +53,35 @@ if Code.ensure_loaded?(Plug) do
       |> send_resp(200, email.html_body)
     end
 
+    get "/:id/attachments/:index" do
+      index = String.to_integer(index)
+      id
+      |> conn.assigns.storage_driver.get()
+      |> case do
+        %{attachments: attachments} when length(attachments) > index ->
+          attachments
+          |> Enum.at(index)
+          |> case do
+            %{data: data, content_type: content_type} when not is_nil(data) ->
```

Comment:
Sorry about than one, I'm a little bit in a hurry today.

Reply:
No worries!